e2e/cypress/integration/messaging/typing_should_show_up_when_editing_spec.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @messaging
+
+describe('Messaging', () => {
+    before(() => {
+        // # Login as test user and visit town-square channel
+        cy.apiInitSetup({loginAfter: true}).then(({team}) => {
+            cy.visit(`/${team.name}/channels/town-square`);
+        });
+    });
+
+    it('M23347 - Typing should show up right away when editing a message using the up arrow', () => {
```

Comment:
Sorry but the ticket is outdated. Please change the key to `MM-T98`
```suggestion
    it('MM-T98 - Typing should show up right away when editing a message using the up arrow', () => {
```